weasyl/controllers/api.py

```diff
@@ -457,6 +462,7 @@ def GET(self):
 
 class api_favorite_(api_base):
     login_required = True
+    oauth_scopes = ['favourite']
```

Comment:
I admire commonwealth English, but 'favorite' is going to be the less surprising spelling so we should use that. In particular, it's the spelling used throughout the site's interface.